src/main/views/crown-daily-list.njk

```diff
@@ -13,26 +13,15 @@
 {% block content %}
   <div class="parent-box">
     <h2 id='page-heading' class="govuk-heading-l">
-      {{ courtName }}: {{ headerPart1 }}
+      {{ title }} for {{ courtName }}
       {% if bill %}
-        <br />/{{ courtName }}: {{ headerPart2 }}
```

Comment:
If headerPart1 and headerPart2 not used anywhere else, they could be removed from the language json files